q2_assembly/quast/quast.py

```diff
@@ -54,10 +55,9 @@ def _process_quast_arg(arg_key, arg_val):
     """
     if isinstance(arg_val, bool) and arg_val:
         return [_construct_param(arg_key)]
-    elif arg_key == "threads" and (not arg_val or arg_val > 1):
-        # TODO: this needs to be fixed (to allow multiprocessing)
+    elif arg_key == "threads" and arg_val > 1 and platform.system() == "Darwin":
```

Comment:
I recommend testing that `platform.system() != 'Linux'` here, so you catch the other possible values for platform.system() (which includes `""`, if it can't be determined) that are likely unsupported.

Reply:
Good point - changed!